packInstaller.ps1

```diff
@@ -0,0 +1,45 @@
+param (
+    [string]$toolPath = ".\build\windows\amd64\ktor.exe",
+    [string]$outPath = "ktor-installer.msi",
+    [string]$wixExe = "wix.exe"
+)
+
+$version = $(git describe --tags --contains --always --abbrev=7)
+
+if (!($version -match '\d+\.\d+.\d+')) {
+    Write-Error "Expected version in the format *.*.*, got ${version}"
+    exit 1
+}
+
+$wixProduct = @"
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+    <Package Name="Ktor CLI" Version="${version}" Manufacturer="JetBrains" UpgradeCode="$(New-Guid)" Scope='perUser'>
+        <MediaTemplate EmbedCab="yes" />
+        <WixVariable Id="WixUILicenseRtf" Value="LICENSE.rtf" />
+
+        <MajorUpgrade
+                AllowDowngrades="no" DowngradeErrorMessage="The newer version of Ktor CLI is already installed"
```

Comment:
Perhaps the text here should state that "The latest version of Ktor CLI is already installed" ? 
Or if not necessarily the latest, it should then be "A newer version..."